.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,56 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thanks for reporting an bug. If you would prefer a private method,
+        please email support@wolfssl.com
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version were you using?
+    validations:
+      required: true
+  - type: textarea
+    id: details
+    attributes:
+      label: description
```

Comment:
For consistency please make this `Description`.

Reply:
Done